data-processing/entities/definitions/commands/detect_definitions.py

```diff
@@ -601,8 +601,8 @@ def process(
             return
 
         # Load the pre-trained definition detection model.
-        prediction_types = ["AI2020", "DocDef2", "W00"]
-        model = DefinitionDetectionModel(prediction_types)
+        prediction_type = "DocDef2+AI2020+W00"
+        model = DefinitionDetectionModel(prediction_type)
```

Comment:
Does it make sense to still provide prediction_type as a pararmeter to DefinitionDetectionModel if only one prediction type will ever be used?

Reply:
This is unfortunately an artefact of how the model is trained. The prediction type text is embedded deep in the model (as the keys to model heads). So, I'm afraid we still need to specify this somewhere.

Reply:
Great, thank you for the clarification. Maybe add this as a comment for DefinitionDetectionModel if it is not yet there?

Reply:
Done!